client/me/notification-settings/settings-form/stream.jsx

```diff
@@ -37,7 +33,7 @@ class NotificationSettingsFormStream extends PureComponent {
 
 		if ( this.props.devices && size( this.props.devices ) > 0 ) {
 			stream = parseInt( this.state.selectedDeviceId || first( this.props.devices ).id, 10 );
-			settings = this.props.settings.find( device => device.get( 'device_id' ) === stream );
+			settings = this.props.settings.find( ( { device_id } ) => device_id === stream );
```

Comment:
`settings = find( settings, { device_id: stream } );`

Reply:
I was very close to pulling in `find` here! Makes sense to have it follow what we see in similar files